CONTRIBUTING.md

```diff
@@ -11,15 +11,18 @@
 - `pnpm lint`
 - `pnpm lint:fix`
 
+## Building the addon
+
+- `pnpm build`
+
 ## Running tests
 
 - `pnpm test` – Runs the test suite on the current Ember version
-- `pnpm test:ember --server` – Runs the test suite in "watch mode"
-- `pnpm test:ember-compatibility` – Runs the test suite against multiple Ember versions
+- `pnpm test:watch` – Runs the test suite in "watch mode"
 
-## Running the dummy application
+## Running the test application
 
 - `pnpm start`
-- Visit the dummy application at [http://localhost:4200](http://localhost:4200).
+- Visit the test application at [http://localhost:4200](http://localhost:4200).
```

Comment:
probably should add `/tests` to this url since going to just the root currently shows nothing 🙈

Reply:
Should we change the blueprints to do that perhaps?

Reply:
well yes, possibly. We need to add a demo app to the blueprint and that will probably prompt us to fix this there. In this current state you will get a 404 if you follow these instructions and that's why I mention it